clientlibs/js/src/functions/markExperimentPoint.ts

```diff
@@ -12,26 +20,22 @@ export default async function markExperimentPoint(
   status: MARKED_DECISION_POINT_STATUS,
```

Comment:
condition is no longer required as we are sending status.
condition is used on the server to check if client has applied the assignment correctly but now we are also sending status so we should remove condition. @danoswaltCL @jerith666